### ISSUE TITLE
MTGO output fallbacks

### ISSUE THREAD

**Issue by nex3** [Senryoku/MTGADraft#352]
First: thanks a bunch for implementing https://github.com/Senryoku/MTGADraft/issues/346! My draft group tried it last week and it's _almost_ perfect, but there's one issue: my cube on CubeCobra includes a few non-MTGO cards in cases where the same promo art exists in both places but with a different card ID, such as [this](https://scryfall.com/card/prm/80943/raugrin-triome) versus [this](https://scryfall.com/card/iko/311/raugrin-triome), since the physical card images simply look a lot better. Unfortunately, this breaks the MTGO export because those cards don't have MTGO IDs!

I doubt it's feasibly to try to match up artwork, but it would be very cool if instead of erroring here, the export just chose an arbitrary MTGO ID for the same card. It's not a big deal if we end up with [this Raugrin Triome](https://scryfall.com/card/iko/251/raugrin-triome) in the decklist and we have to hit "update with versions in my collection" a few times.

**Comment by Senryoku** [comment]
Thanks for the feedback!
I slightly improved the export in b0c77cd, it will check other printings before giving up and even if this also fails, it will still export the rest, warning the user of the missing cards.
I'll try to deploy it faster than last time :)

**Comment by nex3** [comment]
It works—mostly sort of. I've noticed that my decks in particular seem to fail to export... it pops up the "exporting deck..." dialog but I never get a download. I don't see anything obvious going wrong in the console, but I can check again.

One of our players also noticed that Char (which is GK1-28 in my CubeCobra) got downloaded as Abzan Charm instead. There might be a text-inclusion search going on instead of a card name match?

**Comment by Senryoku** [comment]
The second problem should be fixed by [480e038](https://github.com/Senryoku/MTGADraft/commit/480e0388ca7212aee6af868c41e12c5ba771f9ef): I updated the request to use exact names.

Not sure about your own problem, do you have an example deck (in MTGA format :) ) that exhibits the issue? 
No error made me think maybe the requests were simply not going through, so I've added some timeouts to the Scryfall API requests. Maybe you'll have some console output next time.

**Comment by nex3** [comment]
Here we go:

![image](https://user-images.githubusercontent.com/188/209902098-c55afa27-21cf-40f9-993b-61ed1296f1e4.png)

![image](https://user-images.githubusercontent.com/188/209902085-6b99867d-6609-4105-83f1-69f912940c94.png)

<details><summary>MTGA decklist</summary>

```
Deck
1 The Cruelty of Gix (DMU) 87
1 Meren of Clan Nel Toth (CMA) 186
1 Polluted Delta (KTK) 239
1 Nissa, Who Shakes the World (SLD) 518
1 Eternal Witness (MMA) 144
1 Fauna Shaman (BRO) 179
1 Yavimaya Coast (ORI) 252
1 Blood for Bones (M20) 89
1 Castle Locthwain (ELD) 241
1 Overgrown Tomb (RTR) 243
1 Yavimaya Elder (VMA) 240
1 Nullpriest of Oblivion (ZNR) 118
1 Torsten, Founder of Benalia (DMC) 47
1 Verdant Catacombs (ZEN) 229
1 Apex Altisaur (C19) 31
1 Collective Brutality (EMN) 85
1 Biting-Palm Ninja (NEO) 88
1 Sylvan Caryatid (PRM) 50120
1 Devoted Druid (SHM) 110
1 Garruk Wildspeaker (LRW) 213
1 Dig Up (VOW) 197
1 Dread Return (TSP) 104
1 Incubation Druid (RNA) 131
1 Guul Draz Assassin (ROE) 112
1 Garruk Relentless (ISD) 181
1 Song of the Dryads (C14) 47
1 Vivien, Monsters' Advocate (IKO) 175
1 Zagoth Triome (IKO) 313
1 Tear Asunder (DMU) 183
1 Ignoble Hierarch (MH2) 166
1 Green Sun's Zenith (MBS) 81
1 Patron of the Vein (C17) 20
1 Body Double (PLC) 35
1 Utopia Sprawl (A25) 192
1 Slaughter Pact (MMA) 97
1 Pack Rat (RTR) 73
1 Abundant Harvest (STA) 48
1 Temple of Mystery (CP1) 6
1 Swamp
1 Forest

Sideboard
1 Nevinyrral's Disk (MH2) 298
1 Cavalier of Gales (M20) 52
1 Ugin, the Spirit Dragon (FRF) 1
1 Sorin, Grim Nemesis (SOI) 251
1 Elvish Piper (A25) 168
1 Condemn (DIS) 8
1 Timeless Lotus (DMU) 239
5 Plains
5 Island
5 Swamp
5 Mountain
5 Forest
```

</details>

**Comment by Senryoku** [comment]
Thanks for the list. Turns out this was a stupid mistake that went unnoticed only because I was testing in Firefox...

Should be fixed by 7edbbfd029e4dcd50944b5c3b70391423d634ce1.
